pkgs/applications/misc/gollum/default.nix

```diff
@@ -1,37 +1,21 @@
-{ lib, stdenv, bundlerEnv, ruby, makeWrapper, bundlerUpdateScript
-, git, docutils, perl }:
+{ lib, bundlerApp, ruby,
+  makeWrapper, git, docutils }:
 
-stdenv.mkDerivation rec {
+bundlerApp rec {
   pname = "gollum";
-  # nix-shell -p bundix icu zlib cmake pkg-config openssl
-  version = (import ./gemset.nix).gollum.version;
+  exes = [ "gollum" ];
 
-  nativeBuildInputs = [ makeWrapper ];
+  inherit ruby;
+  gemdir = ./.;
 
-  dontUnpack = true;
-
-  installPhase = let
-    env = bundlerEnv {
-      name = "${pname}-${version}-gems";
-      inherit pname ruby;
-      gemdir = ./.;
-    };
-  in ''
-    mkdir -p $out/bin
-    makeWrapper ${env}/bin/gollum $out/bin/gollum \
-      --prefix PATH ":" ${lib.makeBinPath [ git docutils perl]}
-    makeWrapper ${env}/bin/gollum-migrate-tags $out/bin/gollum-migrate-tags \
-      --prefix PATH ":" ${lib.makeBinPath [ git ]}
-  '';
-
-  passthru.updateScript = bundlerUpdateScript "gollum";
```

Comment:
Could you reintroduce `update.sh` as `passthru.updateScript`? That makes it easier to find it, so I can just type `nix shell .#gollum.updateScript`. To me another bonus would be if the script would expect to be run at Nixpkgs root, so one could update the package in one line, by running `nix run .#gollum.updateScript` and one wouldn't need to navigate to the directory.

Reply:
I was not aware of the `passthru.updateScript` mechanism. After little reading about it, I eventually found out [bundlerUpdateScript](https://github.com/NixOS/nixpkgs/blob/902fe7c1034fde45195301ad4d9358206df865f4/pkgs/development/ruby-modules/bundler-update-script/default.nix) does almost exact the same thing as the added `update.sh`. I would for this suggest removing the `update.sh` and restoring this line and using `bundlerUpdateScript` (see ad6f6d4).
The only difference will be to not explicitly setting the environment variable `BUNDLE_FORCE_RUBY_PLATFORM` telling `bundler` to not use architecture optimized gems. But I do not know whether this will cause issues.

Reply:
Sounds good, but now I'm getting an eval error again ;)

```
$ nix shell .#gollum.updateScript
error: undefined variable 'bundlerUpdateScript'

       at /nix/store/zk5d4z6l5znpbxqlaq1j9bbflwyii3ig-source/pkgs/applications/misc/gollum/default.nix:13:27:

           12|
           13|   passthru.updateScript = bundlerUpdateScript "gollum";
             |                           ^
           14|


```

Reply:
^^ sorry. I forgot to include `bundlerUpdateScript` in `default.nix`. I force-pushed f882b180c55de786c21f9dc2c078425e2d01a579 resolving the issue.

Reply:
Thanks, that does the job for me. I'm still unable to do `nix run .#gollum.updateScript`, but that is a problem with `bundlerUpdateScript` in general, so I'll file a separate issue for that.